Documentation/ContentElementRendering/Index.rst

```diff
@@ -0,0 +1,160 @@
+:navigation-title: Content Element rendering
+..  include:: /Includes.rst.txt
+
+..  _content-element-rendering:
+
+====================================================
+Overriding the default templates of content elements
+====================================================
+
+The content elements that are rendered up to this point are rendered by the
+TYPO3 system extension Fluid Styled Content (:composer:`typo3/cms-fluid-styled-content`).
+
+This extensions offers default templates to render content elements. Without
+this such an extension, no content would be rendered at all. The default
+templates provided by this extension can be overridden with site settings
+provided by the Fluid Styled Content.
+
+..  contents::
+
+..  _content-element-rendering-settings:
+
+Use Settings to override template paths of Fluid Styled Content
+===============================================================
+
+Site settings can be saved both in the site configuration and in the site
+package extension.
+
+If you would share your site package across several sites you could set defaults
```

Comment:
I would leave this detail out for a general sitepackage tutorial.

Reply:
Done